cob_bringup/components/cob4_base.launch

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0"?>
+<launch>
+    <arg name="robot"/>
+    <arg name="has_phidget" default="true"/>
+    <arg name="use_bms" default="false"/>
+    <arg name="use_powerstate_phidget" default="$(arg has_phidget)"/>
+    <arg name="sim" default="false"/>
+    
+    <include file="$(find cob_bringup)/components/canopen_base.launch">
+        <arg name="robot" value="$(arg robot)"/>
+        <arg name="sim" value="$(arg sim)"/>
+    </include>
+
+    <group unless="$(arg sim)">
+        <!-- pyhisical parts -->    
+        <include file="$(find cob_bringup)/drivers/sick_s300.launch">
+            <arg name="robot" value="$(arg robot)"/>
+            <arg name="name" value="base_laser_front"/>
+        </include>
+        <include file="$(find cob_bringup)/drivers/sick_s300.launch">
+            <arg name="robot" value="$(arg robot)"/>
+            <arg name="name" value="base_laser_right"/>
+        </include>
+        <include file="$(find cob_bringup)/drivers/sick_s300.launch">
+            <arg name="robot" value="$(arg robot)"/>
+            <arg name="name" value="base_laser_left"/>
+        </include>
+        <include file="$(find cob_bringup)/drivers/sick_flexisoft.launch">
+            <arg name="robot" value="$(arg robot)"/>
+        </include>
+    
+        <include file="$(find cob_bringup)/drivers/light.launch">
+            <arg name="robot" value="$(arg robot)"/>
+            <arg name="component_name" value="light_base"/>
```

Comment:
light.launch has a [sim_enabled](https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_bringup/drivers/light.launch#L7) argument and should be started for simulation, too